tools/ci/ccip_lcov_prune

```diff
@@ -23,8 +22,6 @@ lcov --remove $1 -o $2 \
   'src/v0.8/ccip/libraries/USDPriceWith18Decimals.sol' \
   'src/v0.8/ccip/libraries/MerkleMultiProof.sol' \
   'src/v0.8/ccip/libraries/Pool.sol' \
-  'src/v0.8/ConfirmedOwnerWithProposal.sol' \
-  'src/v0.8/tests/MockV3Aggregator.sol' \
```

Comment:
These no longer live in these places and the shared folder is ignored entirely so no new entry has to be made